app/models/usuario.js

```diff
@@ -9,7 +9,8 @@ module.exports = (sequelize, DataTypes) => {
       password: { type: DataTypes.STRING, allowNull: false }
     },
     {
-      tableName: 'users'
+      tableName: 'users',
+      underscored: true
     }
   );
   return User;
```

Comment:
Change the name of this file

Reply:
I've changed this file name but git does not seem to recognize that this change is in another file.